docs/JH_access.md

```diff
@@ -0,0 +1,54 @@
+
+* Here, you can find all the applications that are enabled on the cluster. Let’s start by clicking on the link for the [JupyterHub application](https://jupyterhub-opf-jupyterhub.apps.smaug.na.operate-first.cloud/).
+
+* This link will redirect you to the login page, where you can choose the `operate-first` login option and you can log in using your Github account.
+
+    ![Operate First Login Page](./../public/assets/Setup_Environment/opf_login.png)
+
+* After successful login, you will be redirected to the jupyterhub spawner page where you can customize the type of jupyter instance you would want to use. This will provide you with a selection of environments, each with some set of pre-installed packages and dependencies, like TensorFlow, Spark, Pandas, etc.
+
+    ![JH Notebook Server](./../public/assets/Setup_Environment/notebook_server.png)
+
+* In addition to the image, you can choose the memory and CPU sizes needed for your task,  there are options for small, medium, or large containers. 
+
+    ![Container Sizes](./../public/assets/Setup_Environment/container_size.png)
+
+* Finally, you can set environment variables that can be used to pass specific information, like API credentials, directly into the jupyterhub environment. This allows you to share the notebooks and ensure that no credentials are exposed on GitHub. For example, if your notebook reads data from object storage, you likely don’t want to explicitly put the access key and secret directly into your notebook.  
+
+    ![Environment Variables](./../public/assets/Setup_Environment/env_variables.png)
+
+* Once you’ve selected the options that are right for you, go ahead and click “Start server”.
+
+* Starting the instance may take a few seconds, up to a minute. While it’s loading, you can examine the event logs to see what’s going on. One thing to note here is that the jupyter server is connecting to a storage volume, which is like your local hard drive, where your notebooks and other files are stored. So as you log on and off or shut down your server completely, your work persists, even if you start a different notebook image later on - it’s really like your own personal drive.
+
+    ![Event logs](./../public/assets/Setup_Environment/event_logs.png)
+
+* When you launch your jupyterlab environment for the first time it will be empty. However, the instance here has connected to the storage volume associated with an existing account and so shows some projects that the user has been working on. This page provides a nice user-friendly UI to navigate the notebooks or other files you have in the remote storage. 
+
+    ![JH start page](./../public/assets/Setup_Environment/jh_start_page.png)
+
+* Now assuming your directory is completely empty, let’s go ahead and create a new notebook. We can click “+” in the upper left-hand corner, then select python3 from the notebook section. This will spawn a new empty “untitled” notebook for us.
```

Comment:
Let's update this instruction for jupyter lab.

Reply:
Added :+1: